CMakeLists.txt

```diff
@@ -344,6 +344,18 @@ if (CONFIG_PICOLIBC AND NOT CONFIG_PICOLIBC_IO_FLOAT)
   zephyr_compile_options($<$<COMPILE_LANGUAGE:C>:$<TARGET_PROPERTY:compiler,no_printf_return_value>>)
 endif()
 
+if (CONFIG_UBSAN)
```

Comment:
no spaces between if and brackets in cmake

Reply:
Thanks, fixed all of these.